compiler/rustc_typeck/src/check/upvar.rs

```diff
@@ -1760,12 +1760,11 @@ impl<'a, 'tcx> euv::Delegate<'tcx> for InferBorrowKind<'a, 'tcx> {
         self.borrow(assignee_place, diag_expr_id, ty::BorrowKind::MutBorrow);
     }
 }
-
-/// Truncate projections so that following rules are obeyed by the captured `place`:
+/// Truncate `place` so that an `unsafe` block isn't required to capture it.
```

Comment:
Note the removed newline above this doc comment. (Don't worry about it, no need to block this one from landing. A drive-by fix in one of your next PRs is fine).